frontend/src/libs/hooks/use-sidebar-show/use-sidebar-show.hook.ts

```diff
@@ -0,0 +1,20 @@
+import { type SetURLSearchParams } from '#libs/types/types.js';
+
+import { useSearchParams as useSearchParameters } from '../hooks.js';
+
+type UseSidebarStateReturn = {
+  isSidebarShownParameter: boolean;
+  setIsSidebarShown: SetURLSearchParams;
+};
+
+const useSidebarState = (): UseSidebarStateReturn => {
+  const [isSidebarShown, setIsSidebarShown] = useSearchParameters();
+
+  const isSidebarShownParameter =
+    isSidebarShown.get('sidebarMode') === 'show' ||
+    isSidebarShown.get('sidebarMode') === null;
```

Comment:
```suggestion
const useSidebarState = (): UseSidebarStateReturn => {
  const [sidebarMode, setSidebarMode] = useSearchParameters();

  const isSidebarShown =
    sidebarMode.get('sidebarMode') === 'show' ||
    sidebarMode.get('sidebarMode') === null;
    
    const setIsSidebarShown = () => {
      // should switch the value
    }
```

Reply:
@Liza-Veis should i add second func for hiding sidebar?
like:
`const useSidebarState = (): UseSidebarStateReturn => {
  const [sidebarMode, setSidebarMode] = useSearchParameters();
  const isSidebarShown =
    sidebarMode.get('sidebarMode') === 'show' ||
    sidebarMode.get('sidebarMode') === null;
    
    const setSidebarShown = () => {
      // should switch the value
    }
        const setSidebarHide = () => {
      // should switch the value
    }`
    or i should pass the value to setIsSidebarShown?

Reply:
setSidebarShown should accept true/false value and set sidebar mode, we don't need setSidebarHide